Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -58,6 +58,7 @@ GetCpuStepping(
   return ((CPU_STEPPING) (Eax.Uint32 & CPUID_FULL_STEPPING));
 }
 
+#if FeaturePcdGet (PcdTccEnabled)
```

Comment:
This condition is a duplicate. Same condition already exists when it is being called.

Reply:
It was added to fix build errors caused due to Tcc FSP UPDs which are used in function definition.